backend/lambda-functions/journeys/handlers/handlers.go

```diff
@@ -260,3 +262,50 @@ func (h *handlers) GetJourney(ctx context.Context, input utils.JourneyInput) (*u
 		Count:     len(steps),
 	}, nil
 }
+
+func (h *handlers) UpdateNormalnessScores(ctx context.Context) error {
+	err := h.db.Transaction(func(tx *gorm.DB) error {
+		err := tx.Exec(fmt.Sprintf("SET LOCAL statement_timeout TO %d", normalnessTimeout)).Error
+		if err != nil {
+			return err
+		}
+
+		return tx.Exec(`
+			with frequencies as (
+				select project_id, url, next_url, count(*)
+				from user_journey_steps
+				where created_at > now() - interval '30 days'
+				group by 1, 2, 3),
```

Comment:
do we need a new index to group on project_id, url, and next_url?

Reply:
good point, already ran
```
CREATE INDEX concurrently idx_user_journey_steps_project_id_c
 reated_at_v3 ON public.user_journey_steps USING btree (project_id, cre
 ated_at, url, next_url)
 ```
 in prod, will add to the gorm model